src/domain/ftps_server.py

```diff
@@ -85,6 +86,23 @@ def run(self):
             self.server.serve_forever()
         self.run_finished.emit()
 
+    #TODO: remove once test is completed
+    def run_op_mode(self):
+        """ TEST Method to create new thread for FTPS Server class."""
+
+        if self.server:
+            thread = threading.Thread(target=self.server.serve_forever())
```

Comment:
is self.server.serve_forever() giving back the callback that Thread should use? If not and you want Thread to use it as callback, then remove the ().

Reply:
This was actually the issue!!!!!